src/content/2025-04-10-osc-at-wicse-codeathon.md

```diff
@@ -0,0 +1,26 @@
+---
+slug: "/2025-wicse-codeathon"
+date: "2025-04-10"
+title: "OSC at WICSE Code-a-thon"
+author: ["Nicolas Valiente"]
+featuredImage: ../images/blog/2025-04-10-wicse-codeathon-workshop.jpeg
+subtitle: Educating the youth to become future FAANG front-end developers
+---
+
+On April 29th, 2025, the Women in Computer Science and Engineering (WICSE) held their annual Code-a-thon. This event marks the day of the year where middle and high schoolers are invited onto UF campus to be exposed to a variety of computing topics. From quantum computing to cybersecurity, there was no shortage of topics to learn. To support WICSE's endeavor and to provide the kids with an incredible experience, high-profile OSC members enlisted to attend and host the most essential workshop...
+
+## **Introduction to HTML & CSS!**
+
+OSC's workshop of valuable front-end technologies had one goal and one plan. The goal: turn the kids into incoming software engineers at Google. The plan: explain what HTML and CSS even is, and have them make their own website with nothing but a computer, VS Code, and pure imagination. The scope of the workshop was kept small, only introducing barebone-yet-essential elements of HTML, and a very brief introduction into how styling works with CSS. A delicate balance had to be made, ensuring that the workshop taught enough without going too far with concepts so that kids with no programming experience can follow along.
```

Comment:
“The scope of the workshop was kept small”

No! It was MASSIVE. Dont underplay OSCs glorious achievements.